tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -2635,6 +2635,11 @@ Status AddFusedContractionNode(
   const NodeDef& bias_add = graph->node(matched.bias_add);
   fused_conv.add_input(bias_add.input(matched.bias_port));  // 2: bias
 
+  // Add OP has two inputs, one is conv+bias pattern matched previously,
+  // the other input to add is fused here.
+  const NodeDef& add = graph->node(matched.add);
+  fused_conv.add_input(add.input(1 - matched.port_id));
```

Comment:
Could you please explain why we need to change the order here?

Reply:
I need to move the add_input to before calling the CopyConv2DAttributes due to the change from the commit 5be9a59, specifically, the following line requires all the add_input has been called to get the correct input_size. 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/grappler/optimizers/remapper.cc#L2139

Thanks for the quick review.